lib/vmdb/loggers.rb

```diff
@@ -39,7 +39,7 @@ def self.apply_config(config)
     private
 
     def self.create_loggers
-      if ENV.key?("CI")
+      if false && ENV.key?("CI")
```

Comment:
This should have a comment explaining why this was done.  Note, if there is little to no speed difference with this change, I'd be in favor of removing this conditional entirely.